templates/strings-dot-syntax-swift3.stencil

```diff
@@ -10,12 +10,14 @@ import Foundation
 // swiftlint:disable nesting
 // swiftlint:disable variable_name
 // swiftlint:disable valid_docs
+{% macro parametersBlock params %}{% for type in params.types %}_ p{{forloop.counter}}: {{type}}{% if not forloop.last %}, {% endif %}{% endfor %}{% endmacro %}
+{% macro argumentsBlock params %}{% for type in params.types %}p{{forloop.counter}}{% if not forloop.last %}, {% endif %}{% endfor %}{% endmacro %}
```

Comment:
Ah, right, sorry I talked too fast, was thinking about `params.types|join(", ")` but that's not what you want, you don't want `Int, Float, String` but `p0, p1, p2` here, right. nvm